server/controller/cloud/aws/vinterface_and_ip.go

```diff
@@ -63,6 +63,18 @@ func (a *Aws) getVInterfacesAndIPs(region awsRegion) ([]model.VInterface, []mode
 	// types.NetworkInterface.Description: aws-K8S-i-01994fbd5e2d8xxxx
 	eksNodeInstanceIDs := map[string]bool{}
 	for _, vData := range retVinterfaces {
+		// get it from the Instance ID tag first
```

Comment:
thanks for your contribution.
As with your code, the node id is initially obtained from the tag, but the tag can be modified and cannot be fully trusted, so the interface description is used to obtain it.

Reply:
In the real cases, many of the description of EKS nodes are empty, and more EKS node can be obtained by using the instance ID tag. Yes, the tag my be modified, any data may be modified, but we tend to get more data from the perspective of data integrity, of course, this is just a suggestion, it's up to you